streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopicManager.java

```diff
@@ -100,13 +100,19 @@ public InternalTopicManager(final Admin adminClient, final StreamsConfig streams
         final Set<String> newlyCreatedTopics = new HashSet<>();
 
         while (!topicsNotReady.isEmpty() && remainingRetries >= 0) {
-            topicsNotReady = validateTopics(topicsNotReady, topics);
+            final Set<String> tempUnknownTopics = new HashSet<>();
```

Comment:
Better to be name as `leaderUnavailableTopics` to match the error code.

Reply:
Well, this variable naming is actually suggested by @ableegoldman , and I also think the `tempUnknownTopics` is more descriptive. Is that OK for you? 

https://github.com/apache/kafka/pull/8712#discussion_r444555724

![image](https://user-images.githubusercontent.com/43372967/86199833-b6cea400-bb8d-11ea-81b5-7ba5b6064709.png)

Reply:
Sure, I don't feel strong about reverting the naming once we already got a reason there.